hopsworks-web/yo/app/scripts/controllers/newJobCtrl.js

```diff
@@ -801,8 +813,40 @@ angular.module('hopsWorksApp')
             this.selectFile = function (reason, parameter) {
               ModalService.selectFile('lg',  self.projectId,  self.selectFileRegexes[reason],
                       self.selectFileErrorMsgs["PYSPARK"], false).then(
-                      function (success) {
-                        self.onFileSelected(reason, success);
+                      function (path) {
+                        self.jobConfigFromMetadata = null;
+                        //get attached jupyter configuration. For ipynb and files in Jupyter folder
+                        if(path.includes(".ipynb") && path.includes("/Jupyter")) {
```

Comment:
.ipynb files can be in other folders than "Jupyter". Can we remove this part of the if statement?